src/mumble/MainWindow.cpp

```diff
@@ -95,7 +95,11 @@ OpenURLEvent::OpenURLEvent(QUrl u) : QEvent(static_cast< QEvent::Type >(OU_QEVEN
 	url = u;
 }
 
-MainWindow::MainWindow(QWidget *p) : QMainWindow(p) {
+MainWindow::MainWindow(QWidget *p)
+	: QMainWindow(p),
+	  m_localVolumeLabel(new MenuLabel(tr("Local Volume Adjustment:"), this), Mumble::QtUtils::deleteQObject),
```

Comment:
Use `std::make_unique` instead of invoking `new` explicitly

Reply:
make_unique does not work with custom deleters